### ISSUE TITLE
[BUG] Cannot delete at index with multi-frequency file.

### ISSUE THREAD

**Issue by eurunuela** [physiopy/phys2bids#141]
## Expected Behavior

The`.delete_at_index(idx)` function should delete at the specified index even with multi-frequency files.

## Actual Behavior

The function gives the following error:

```
phys2bids -in Test_belt_pulse_multifreq -chtrig 0 -debug
Traceback (most recent call last):
  File "/usr/local/bin/phys2bids", line 11, in <module>
    load_entry_point('phys2bids', 'console_scripts', 'phys2bids')()
  File "/Users/enekourunuela/phys2bids/phys2bids/phys2bids.py", line 300, in _main
    phys_out[uniq_freq].delete_at_index(phys_in.ch_amount - i - 1)
  File "/Users/enekourunuela/phys2bids/phys2bids/physio_obj.py", line 235, in delete_at_index
    del self.timeseries[idx]
TypeError: list indices must be integers or slices, not float
```

## Steps to Reproduce the Problem

    1. Run the following command `phys2bids -in Test_belt_pulse_multifreq -chtrig 0 -debug` using the test file provided by the package.

## Specifications

    - Python version: 3.7.6
    - phys2bids version: v1.1.0-beta+288.gaeb4cc4.dirty
    - Platform: MacOS

**Comment by smoia** [comment]
Was this solved by #164 ?

**Comment by vinferrer** [comment]
Yep I did

**Comment by smoia** [comment]
This was solved by #164
